docs/tutorials/notebooks/EntropicMaps_Barriere_Peltier.ipynb

```diff
@@ -0,0 +1,1562 @@
+{
```

Comment:
<p>typos:</p><p>"based on the results of presented in"</p><p>"We considere"</p><p>"given the two probability distributions P and Q," -&gt; given two, P, Q should be written in markdown</p><p>REF</p><p> </p><p>idea: is the idea really to let \varepsilon go to 0?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>